### ISSUE TITLE
dark mode doesn't work during daylight hours?

### ISSUE THREAD

**Issue by davedean** [davedean/tdcm#15]
must be setting to "auto" instead of "dark"?

**Comment by eavesr** [comment]
Seems like on phone it strictly follows your phone theme and can't be changed. Will investigate - was working fine previously...

**Comment by davedean** [comment]
Yeah, I must have broken it slicing up the page into react components further (to add logout button). I'll get it working again, dark mode rocks.

**Comment by davedean** [comment]
just worked with chatgpt on fixing this, updated and pushed to main, should be working now (and I learnt more!)

**Comment by davedean** [comment]
I think auto is broken now, leaving this open so I remember to fix
